libsbp_ros_msgs/include/libsbp_ros_msgs/ros_conversion.h

```diff
@@ -4,7 +4,7 @@
 #include <libsbp/navigation.h>
 #include <ros/assert.h>
 #include <ros/time.h>
-#include <Eigen/Dense>
+#include <eigen3/Eigen/Dense>
```

Comment:
interesting - did you run into issues building without `eigen3` in the include path? Just curious - not blocking

Reply:
yeah, i faced compilation issues without `eigen3` in the path

Reply:
Yeah this seems odd...maybe you're missing something in the [CMakeLists.txt](https://github.com/Humatics/ethz_piksi_ros/blob/HIT-244/libsbp_ros_msgs/CMakeLists.txt)? 

Usually there's a  `include_directories(${EIGEN3_INCLUDE_DIRS})` or `target_include_directories(...)`

Reply:
@ssonawane-humatics have you tried appending the mentioned line and reverting to the original #include line? does that build for you?

Reply:
Yes, that change has been included in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/5b9212ab5bc5590b3dc0c3fa519b1217ea3074da